### PR TITLE
Fix Python 3 map handling.

### PR DESCRIPTION
**Pull request by hellysmile**

### CODE REVIEW

**Comment by ajdavis** [comment]
Interesting, thanks.

**Comment by hellysmile** [comment]
Any chance to release this hotfix?

**Comment by ajdavis** [comment]
Obviously https://pypi.python.org/pypi/motor/0.6.2

File: motor/motor_tornado.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 from .metaprogramming import create_class_with_framework
 
 # See https://bugs.python.org/issue21720
-__all__ = map(str, ['MotorClient', 'MotorReplicaSetClient', 'Op'])
+__all__ = list(map(str, ['MotorClient', 'MotorReplicaSetClient', 'Op']))
```
Comment (drgarcia1986) [inline motor/motor_tornado.py]: Why not just?
```python
__all__ = ['MotorClient', 'MotorReplicaSetClient', 'Op']
```

Comment (ajdavis) [inline motor/motor_tornado.py]: See https://bugs.python.org/issue21720

**Pull request merged**